distribution/std-lib/Base/src/Data/Time/Locale.enso

```diff
@@ -0,0 +1,106 @@
+from Base import all
+
+polyglot java import java.util.Locale as JavaLocale
+
+# Constants
+
+## The default locale.
+default : Locale
+default = here.from_java (Polyglot.get_member JavaLocale "ROOT")
+
+## The locale for Canada.
+canada : Locale
+canada = here.from_java (Polyglot.get_member JavaLocale "CANADA")
+
+## The locale for China.
+china : Locale
+china = here.from_java (Polyglot.get_member JavaLocale "CHINA")
+
+## The locale for France.
+france : Locale
+france = here.from_java (Polyglot.get_member JavaLocale "FRANCE")
+
+## The locale for Germany.
+germany : Locale
+germany = here.from_java (Polyglot.get_member JavaLocale "GERMANY")
+
+## The locale for Italy.
+italy : Locale
+italy = here.from_java (Polyglot.get_member JavaLocale "ITALY")
+
+## The locale for Japan.
+japan : Locale
+japan = here.from_java (Polyglot.get_member JavaLocale "JAPAN")
+
+## The locale for Korea.
+korea : Locale
+korea = here.from_java (Polyglot.get_member JavaLocale "KOREA")
+
+## The locale for PRC.
+prc : Locale
+prc = here.from_java (Polyglot.get_member JavaLocale "PRC")
+
+## The locale for Taiwan.
+taiwan : Locale
+taiwan = here.from_java (Polyglot.get_member JavaLocale "TAIWAN")
+
+## The locale for the UK.
+uk : Locale
+uk = here.from_java (Polyglot.get_member JavaLocale "UK")
+
+## The locale for the US.
+us : Locale
+us = here.from_java (Polyglot.get_member JavaLocale "US")
```

Comment:
What's the key to this assortment of countries? Seems pretty strange to me. Misses some large countries (India, Russia, Brazil) while exposing, e.g. Taiwan. I'd go by countries-with-highest-population (or countries with highest population + countries with highest GDP, or some other metric like that)